src/reformat.c

```diff
@@ -1073,6 +1081,32 @@ static avifResult avifImageYUV8ToRGB8Mono(const avifImage * image, avifRGBImage
     return AVIF_RESULT_OK;
 }
 
+union F16 {
+  float f;
+  uint32_t u32;
+};
+
+static avifResult avifRGBImageToF16(avifRGBImage * rgb) {
+    avifResult libyuvResult = avifRGBImageToF16LibYUV(rgb);
+    if (libyuvResult != AVIF_RESULT_NOT_IMPLEMENTED) {
+        return libyuvResult;
+    }
+    const int channelCount = avifRGBFormatChannelCount(rgb->format);
+    const float scale = 1.0f / ((1 << rgb->depth) - 1);
+    const float multiplier = 1.9259299444e-34f * scale;
```

Comment:
It would be good to get Frank Barchard's permission to copy his comments before the `HalfFloatRow_C()` function.

Until then, let's add a link to the libyuv function as the source of this algorithm.

Reply:
Done.